frontend/ui/uimanager.lua

```diff
@@ -701,6 +701,12 @@ function UIManager:sendEvent(event)
                 checked_widgets[widget] = true
                 if widget.widget:handleEvent(event) then return end
             end
+            if widget.widget.modal then
```

Comment:
Seems like it could just be an or?

Reply:
It could be a `or` indeed, but I don't mind the split because of the lengthy comments.
But that `if` might be a `elseif`. Dunno if a `is_always_active` might also be a `modal`, but calling _twice_ handleEvent on a same widget, even if it returns false, might have some side effect (can't think of any right now, but logically, it could).
A `or` would solve this issue too, so why not :)